scripts/config.ts

```diff
@@ -410,14 +410,15 @@ function writeConfigs(argv: any) {
     }
 
     if (argv.espresso && argv.enableCaffNode) {
+      sequencerConfig.node.sequencer = false
+      sequencerConfig.node["delayed-sequencer"].enable = false;
       sequencerConfig.node["parent-chain-reader"].enable = false;
       sequencerConfig.execution.sequencer["enable-caff-node"] = true;
       sequencerConfig.execution.sequencer["caff-node-config"] = {
         "hotshot-urls": [argv.espressoUrl],
         "next-hotshot-block": 1,
         namespace: 412346,
         "parent-chain-node-url": argv.l1url,
-        "sequencer-url": argv.l2url,
```

Comment:
We removed this but we didnt add the forwarding target?  How is this code working lol